main.html.in

```diff
@@ -252,6 +252,8 @@
         <button id="benchmark-startup-baseline">Build Benchmark Baseline</button>
         <label><input type="number" min="1" max="100" id="benchmark-num-rounds"> Rounds</label>
         <label><input type="number" id="benchmark-round-delay" min="0" max="20000"> Round Delay(ms)</label>
+        <label><input type="checkbox" id="benchmark-warm-startup">Benchmark warm startup</label>
+        <label><input type="number" id="benchmark-startfg-delay" min="0" max="20000"> Delay after starting BG MIDlet</label>
```

Comment:
It isn't clear why we'll want to change it; perhaps to accommodate a slow device?